app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -38,8 +38,10 @@ def web_console_button(host, *args)
     end
 
     def host_title_actions(*args)
-      title_actions(button_group(schedule_job_multi_button(*args)),
-        button_group(web_console_button(*args)))
+      if authorized_for(permission: 'create_job_invocations', auth_object: args.first)
+        title_actions(button_group(schedule_job_multi_button(*args)),
+          button_group(web_console_button(*args)))
```

Comment:
I don't think this permission is needed to access the web console

Reply:
Fixed it, it seems both button_group and title_actions are handling nil, quite ok :)